tests/src/ContextVeilTest.php

```diff
@@ -53,10 +53,6 @@ public function testReturnsOriginMethodResults(): void
      */
     public function testReturnsUpdatedOriginMethodResults(): void
     {
-        /**
-         * @todo #94:30min Add a way to provide one several contexts for
-         *  different methods.
-         */
```

Comment:
Marker is useless because the functionality was already implemented in a different PR.